### PR TITLE
- REST API call examples for Policies and Watches

### PR DESCRIPTION
**Pull request by danielmkn**
- updated env file

### CODE REVIEW

**Review by chb0github — DISMISSED** [review]

**Comment by chb0github** [comment]
you can add a `http-client.env.priv.json` to contain attributes - it is `.gitignore`d so you get your own private copy

File: http/http-client.env.json [hunk 1]
```diff
@@ -1,8 +1,11 @@
 {
-  "localhost" : {
-    "host" : "http://localhost:9000"
-  },
-  "partnership": {
-    "host" : "https://partnership.jfrog.io"
+  "dev": {
+    "host": "http://licalhost:8081",
```
Comment (alexhung) [inline http/http-client.env.json:3]: Typo

**Review by alexhung — CHANGES_REQUESTED** [review]
LGTM except one typo in the host value

File: http/http-client.env.json (hunk 1 above)
Comment (danielmkn) [reply http/http-client.env.json]: thanks, fixed

**Review by alexhung — APPROVED** [review]

**Pull request merged**